ml-libs/CMakeLists.txt

```diff
@@ -6,6 +6,54 @@ if(THEROCK_ENABLE_MIOPEN)
     list(APPEND optional_profiler_deps roctracer rocprofiler-sdk)
   endif()
 
+  ##############################################################################
+  # Composable_kernel
+  ##############################################################################
+
+  therock_cmake_subproject_declare(composable_kernel
+    EXTERNAL_SOURCE_DIR "composable_kernel"
+    BACKGROUND_BUILD
+    CMAKE_ARGS
+      -DHIP_PLATFORM=amd
+      -DROCM_PATH=
+      -DROCM_DIR=
+      "-DBUILD_TESTING=${THEROCK_BUILD_TESTING}"
+      -DMIOPEN_REQ_LIBS_ONLY=ON
+      -DCMAKE_BUILD_TYPE=Release
```

Comment:
Can probably omit the build type here. Other subprojects inherit the build type from the superproject, unless an override build type is set, like here: https://github.com/ROCm/TheRock/blob/b3af08a5cc0eceb589d02291585e4c4d46c2613d/CMakePresets.json#L15-L18

If CK _requires_ Release (e.g. it fails to build or is too slow to build for CI/CD in RelWithDebInfo), we could edit that preset file.